dashboards/lodestar_execution_engine.json

```diff
@@ -56,613 +56,1405 @@
         "y": 0
       },
       "id": 25,
-      "panels": [],
-      "targets": [
+      "panels": [
```

Comment:
@tuyennhv the large diff was result of moving into the `panels` array here and on L1479 (old 1431)